extensions/amp-sticky-ad/1.0/validator-amp-sticky-ad.protoascii

```diff
@@ -0,0 +1,46 @@
+#
+# Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+tags: {  # <script custom-element="amp-sticky-ad">
+  html_format: AMP  # Ads are not allowed inside ads.
+  tag_name: "SCRIPT"
+  satisfies: "amp-sticky-ad extension .js script"
```

Comment:
In PR #9742 `requires`/`satisfies` has been changed in favor of using `requires_extension`. See https://github.com/ampproject/amphtml/blob/master/extensions/amp-sticky-ad/0.1/validator-amp-sticky-ad.protoascii